injector/src/main/java/edu/ucr/cs/riple/injector/changes/RemoveAnnotation.java

```diff
@@ -64,6 +64,7 @@ public Modification visit(NodeWithAnnotations<?> node, Range range) {
   }
 
   @Override
+  @SuppressWarnings("unchecked")
```

Comment:
Again why suppress

Reply:
@msridhar The only way to resolve this warning is to change the json dependency to `org.json` from `com.googlecode.json-simple:json-simple` since it is compiled with a very old javac (the latest release is for 2012). I am definitely interested in updating this dependency to use `org.json` but that will include a lot of changes which makes this PR pretty large. Should I do it in a follow up PR with an issue for now to track it ?

Reply:
Just created a followup PR that removes the added `@SuppressWarnings` in this PR and all existing ones by changing dependency to `org.json`. #123.